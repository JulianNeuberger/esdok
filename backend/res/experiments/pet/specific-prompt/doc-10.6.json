{"text": "The MSPN sents a dismissal to the MSPO . The MSPO reviews the dismissal . The MSPO rejects the dismissal of the MSPN or The MSPO confirms the dismissal of the MSPN .", "name": "doc-10.6", "id": "doc-10.6", "category": "", "tokens": [{"text": "The", "indexInDocument": 0, "posTag": "DT", "sentenceIndex": 0}, {"text": "MSPN", "indexInDocument": 1, "posTag": "NNP", "sentenceIndex": 0}, {"text": "sents", "indexInDocument": 2, "posTag": "VBZ", "sentenceIndex": 0}, {"text": "a", "indexInDocument": 3, "posTag": "DT", "sentenceIndex": 0}, {"text": "dismissal", "indexInDocument": 4, "posTag": "NN", "sentenceIndex": 0}, {"text": "to", "indexInDocument": 5, "posTag": "TO", "sentenceIndex": 0}, {"text": "the", "indexInDocument": 6, "posTag": "DT", "sentenceIndex": 0}, {"text": "MSPO", "indexInDocument": 7, "posTag": "NNP", "sentenceIndex": 0}, {"text": ".", "indexInDocument": 8, "posTag": ".", "sentenceIndex": 0}, {"text": "The", "indexInDocument": 9, "posTag": "DT", "sentenceIndex": 1}, {"text": "MSPO", "indexInDocument": 10, "posTag": "NNP", "sentenceIndex": 1}, {"text": "reviews", "indexInDocument": 11, "posTag": "VBZ", "sentenceIndex": 1}, {"text": "the", "indexInDocument": 12, "posTag": "DT", "sentenceIndex": 1}, {"text": "dismissal", "indexInDocument": 13, "posTag": "NN", "sentenceIndex": 1}, {"text": ".", "indexInDocument": 14, "posTag": ".", "sentenceIndex": 1}, {"text": "The", "indexInDocument": 15, "posTag": "DT", "sentenceIndex": 2}, {"text": "MSPO", "indexInDocument": 16, "posTag": "NNP", "sentenceIndex": 2}, {"text": "rejects", "indexInDocument": 17, "posTag": "VBZ", "sentenceIndex": 2}, {"text": "the", "indexInDocument": 18, "posTag": "DT", "sentenceIndex": 2}, {"text": "dismissal", "indexInDocument": 19, "posTag": "NN", "sentenceIndex": 2}, {"text": "of", "indexInDocument": 20, "posTag": "IN", "sentenceIndex": 2}, {"text": "the", "indexInDocument": 21, "posTag": "DT", "sentenceIndex": 2}, {"text": "MSPN", "indexInDocument": 22, "posTag": "NNP", "sentenceIndex": 2}, {"text": "or", "indexInDocument": 23, "posTag": "CC", "sentenceIndex": 2}, {"text": "The", "indexInDocument": 24, "posTag": "DT", "sentenceIndex": 2}, {"text": "MSPO", "indexInDocument": 25, "posTag": "NNP", "sentenceIndex": 2}, {"text": "confirms", "indexInDocument": 26, "posTag": "VBZ", "sentenceIndex": 2}, {"text": "the", "indexInDocument": 27, "posTag": "DT", "sentenceIndex": 2}, {"text": "dismissal", "indexInDocument": 28, "posTag": "NN", "sentenceIndex": 2}, {"text": "of", "indexInDocument": 29, "posTag": "IN", "sentenceIndex": 2}, {"text": "the", "indexInDocument": 30, "posTag": "DT", "sentenceIndex": 2}, {"text": "MSPN", "indexInDocument": 31, "posTag": "NNP", "sentenceIndex": 2}, {"text": ".", "indexInDocument": 32, "posTag": ".", "sentenceIndex": 2}], "mentions": [{"type": "activity", "tokenDocumentIndices": [2, 3, 4]}, {"type": "activity", "tokenDocumentIndices": [11]}, {"type": "activity", "tokenDocumentIndices": [17]}, {"type": "activity", "tokenDocumentIndices": [26]}, {"type": "actor", "tokenDocumentIndices": [0, 1]}, {"type": "actor", "tokenDocumentIndices": [9, 10]}, {"type": "actor", "tokenDocumentIndices": [15, 16]}, {"type": "actor", "tokenDocumentIndices": [24, 25]}, {"type": "actor", "tokenDocumentIndices": [21, 22]}, {"type": "actor", "tokenDocumentIndices": [30, 31]}, {"type": "activity data", "tokenDocumentIndices": [12, 13]}, {"type": "activity data", "tokenDocumentIndices": [18, 19, 20, 21, 22]}, {"type": "activity data", "tokenDocumentIndices": [27, 28, 29, 30, 31]}, {"type": "xor gateway", "tokenDocumentIndices": [23]}, {"type": "condition specification", "tokenDocumentIndices": [18, 19, 20, 21, 22]}, {"type": "condition specification", "tokenDocumentIndices": [27, 28, 29, 30, 31]}], "entities": [{"mentionIndices": [4, 8, 9]}, {"mentionIndices": [5, 6, 7]}, {"mentionIndices": [10, 11, 12]}, {"mentionIndices": [0]}, {"mentionIndices": [1]}, {"mentionIndices": [2]}, {"mentionIndices": [3]}, {"mentionIndices": [13]}, {"mentionIndices": [14]}, {"mentionIndices": [15]}], "relations": [{"headMentionIndex": 13, "tailMentionIndex": 15, "type": "same gateway"}, {"headMentionIndex": 0, "tailMentionIndex": 1, "type": "flow"}, {"headMentionIndex": 1, "tailMentionIndex": 2, "type": "flow"}, {"headMentionIndex": 2, "tailMentionIndex": 14, "type": "flow"}, {"headMentionIndex": 14, "tailMentionIndex": 13, "type": "flow"}, {"headMentionIndex": 13, "tailMentionIndex": 3, "type": "flow"}, {"headMentionIndex": 3, "tailMentionIndex": 15, "type": "flow"}, {"headMentionIndex": 0, "tailMentionIndex": 4, "type": "actor performer"}, {"headMentionIndex": 0, "tailMentionIndex": 5, "type": "actor recipient"}, {"headMentionIndex": 1, "tailMentionIndex": 5, "type": "actor performer"}, {"headMentionIndex": 1, "tailMentionIndex": 10, "type": "uses"}, {"headMentionIndex": 2, "tailMentionIndex": 6, "type": "actor performer"}, {"headMentionIndex": 2, "tailMentionIndex": 11, "type": "uses"}, {"headMentionIndex": 3, "tailMentionIndex": 7, "type": "actor performer"}, {"headMentionIndex": 3, "tailMentionIndex": 12, "type": "uses"}]}