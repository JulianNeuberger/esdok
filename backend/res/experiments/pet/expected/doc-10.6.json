{"nodes": [{"id": "0", "name": "The MSPN", "entity": {"name": "actor", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.6", "pageStart": 1, "pageEnd": 1}}, {"id": "1", "name": "sents", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.6", "pageStart": 1, "pageEnd": 1}}, {"id": "2", "name": "the dismissal of the MSPN", "entity": {"name": "activity data", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.6", "pageStart": 1, "pageEnd": 1}}, {"id": "3", "name": "the MSPO", "entity": {"name": "actor", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.6", "pageStart": 1, "pageEnd": 1}}, {"id": "4", "name": "reviews", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.6", "pageStart": 1, "pageEnd": 1}}, {"id": "5", "name": "rejects", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.6", "pageStart": 1, "pageEnd": 1}}, {"id": "6", "name": "or", "entity": {"name": "xor gateway", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.6", "pageStart": 1, "pageEnd": 1}}, {"id": "7", "name": "confirms", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.6", "pageStart": 1, "pageEnd": 1}}], "edges": [{"id": "0", "source": {"id": "1", "name": "sents", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.6", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "2", "name": "the dismissal of the MSPN", "entity": {"name": "activity data", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.6", "pageStart": 1, "pageEnd": 1}}, "type": "uses"}, {"id": "1", "source": {"id": "1", "name": "sents", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.6", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "0", "name": "The MSPN", "entity": {"name": "actor", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.6", "pageStart": 1, "pageEnd": 1}}, "type": "actor performer"}, {"id": "2", "source": {"id": "1", "name": "sents", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.6", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "3", "name": "the MSPO", "entity": {"name": "actor", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.6", "pageStart": 1, "pageEnd": 1}}, "type": "actor recipient"}, {"id": "3", "source": {"id": "1", "name": "sents", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.6", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "4", "name": "reviews", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.6", "pageStart": 1, "pageEnd": 1}}, "type": "flow"}, {"id": "4", "source": {"id": "4", "name": "reviews", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.6", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "2", "name": "the dismissal of the MSPN", "entity": {"name": "activity data", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.6", "pageStart": 1, "pageEnd": 1}}, "type": "uses"}, {"id": "5", "source": {"id": "4", "name": "reviews", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.6", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "3", "name": "the MSPO", "entity": {"name": "actor", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.6", "pageStart": 1, "pageEnd": 1}}, "type": "actor performer"}, {"id": "6", "source": {"id": "4", "name": "reviews", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.6", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "6", "name": "or", "entity": {"name": "xor gateway", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.6", "pageStart": 1, "pageEnd": 1}}, "type": "flow"}, {"id": "7", "source": {"id": "5", "name": "rejects", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.6", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "2", "name": "the dismissal of the MSPN", "entity": {"name": "activity data", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.6", "pageStart": 1, "pageEnd": 1}}, "type": "uses"}, {"id": "8", "source": {"id": "5", "name": "rejects", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.6", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "3", "name": "the MSPO", "entity": {"name": "actor", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.6", "pageStart": 1, "pageEnd": 1}}, "type": "actor performer"}, {"id": "9", "source": {"id": "6", "name": "or", "entity": {"name": "xor gateway", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.6", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "7", "name": "confirms", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.6", "pageStart": 1, "pageEnd": 1}}, "type": "flow"}, {"id": "10", "source": {"id": "6", "name": "or", "entity": {"name": "xor gateway", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.6", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "5", "name": "rejects", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.6", "pageStart": 1, "pageEnd": 1}}, "type": "flow"}, {"id": "11", "source": {"id": "7", "name": "confirms", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.6", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "2", "name": "the dismissal of the MSPN", "entity": {"name": "activity data", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.6", "pageStart": 1, "pageEnd": 1}}, "type": "uses"}, {"id": "12", "source": {"id": "7", "name": "confirms", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.6", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "3", "name": "the MSPO", "entity": {"name": "actor", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.6", "pageStart": 1, "pageEnd": 1}}, "type": "actor performer"}]}